src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-server.port=8081
```

Comment:
Why did you change the server port?

Reply:
Sorry it got struck for .. it can be changed

Reply:
it got struck in local run due to local port binding issues... so modified temporarily ..

Reply:
this happend to me but i put  this cmd  `netstat -ano | findstr :yourPortNumber` and kill  the process with `taskkill /PID typeyourPIDhere /F`